pkgs/by-name/re/refine/package.nix

```
@@ -0,0 +1,66 @@
+{
+  lib,
+  fetchFromGitLab,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  python3,
+  libxml2,
+  python3Packages,
+  libportal,
+  libportal-gtk4,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "refine";
+  version = "0.2.0";
+  pyproject = false; # uses meson
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "TheEvilSkeleton";
+    repo = "Refine";
+    tag = version;
+    hash = "sha256-NmfsogXkJicZj/cE9HFz91ERD9fxOGKoBV4h68KIE+g=";
+  };
+
+  nativeBuildInputs = [
+    blueprint-compiler
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook4
+    python3
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    libxml2
+    libadwaita
+  ];
+
+  dependencies =
+    [
+      libportal
+      libportal-gtk4
+    ]
+    ++ (with python3Packages; [
+      pygobject3
+    ]);
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Tweak various aspects of GNOME";
+    homepage = "https://gitlab.gnome.org/TheEvilSkeleton/Refine";
+    mainProgram = "refine";
+    platforms = lib.platforms.linux;
+    license = with lib.licenses; [ gpl3Plus ];
+    maintainers = with lib.maintainers; [ aucub ];
```

```suggestion
    maintainers = with lib.maintainers; [
      aucub
      getchoo
    ];
```
Feel free to add me as well :)